contracts/PushComm/PushCommV3.sol

```diff
@@ -591,10 +594,7 @@ contract PushCommV3 is Initializable, PushCommStorageV2, IPushCommV3, PausableUp
             if (amount == 0) {
```

Comment:
The if blocks in external  `createCrossChainRequest` function only check for those calls where the amount needs to be checked. 

In case of deactivateChannel requests there are no checks in this function. So the payload is directly created and sent to the internal  `_createCrossChainRequest` function. Where it is being checked whether the amount is zero or not, if it is zero then we don't call the NTT transfer function, if it is non-zero then we call the transfer function.